### DISCUSSION TITLE
Deleting a schedule

### DISCUSSION THREAD

**Discussion by Ch00k** [django-q2/django-q2#249] [General]
For my use case I need to be able to create and delete arbitrary schedules programmatically. Looking at the function `django_q.tasks.schedule` it looks like in order to delete a schedule one would need to simply do `Schedule.objects.get(id="foo").delete()`. Is this all that's required, or is there anything else that I need to take care of if I want to delete a schedule?

Would it perhaps be nice to have a `django_q.tasks.delete_schedule` function for cases when you need to delete a schedule from code (as opposed to deleting from Django Admin)?

**Comment by GDay** [comment]
You can just delete it. It shouldn't trigger again after that.
